content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -2793,6 +2800,124 @@ Bans the customer for a specific period of time. It immediately disconnects all
 <Section>
 <Text>
 
+### Follow Customer
+
+Marks a customer as followed. As a result, the requester (an agent) will receive the info about all the changes related to that customer via pushes. Once the customer leaves the website or is [unfollowed](#unfollow-customer), the agent will no longer receive that information. 
```

Comment:
could you copy-paste it to the Web reference as well?

Reply:
done